pcdsdevices/interface.py

```diff
@@ -552,7 +549,7 @@ class MvInterface(BaseInterface):
     applications.
     """
 
-    tab_whitelist = ["mv", "wm", "camonitor", "wm_update"]
```

Comment:
I personally never use it, but my understanding was `camonitor` was an "old hutch Python thing that was really desirable". No longer the case?

Reply:
I think wm_update does exactly the same thing